### PR TITLE
Code blocks --> Final block of code blocks

### PR DESCRIPTION
**Pull request by kodster28**
### Summary

Final bits of work to remove most `$` from our code blocks

### CODE REVIEW

**Comment by kodster28** [comment]
Relates to #16210

**Review by haleycode — APPROVED** [review]

File: src/content/docs/workers/tutorials/connect-to-turso-using-workers/index.mdx [hunk 1]
```diff
@@ -128,11 +146,11 @@ The Turso client library requires two pieces of information to make a connection
 
 To get the URL for your database, run the following Turso CLI command, and copy the result:
 
-```sh
-$ turso db show my-db --url
-# Output:
+````sh
```
Comment (kodster28) [inline src/content/docs/workers/tutorials/connect-to-turso-using-workers/index.mdx]:
```suggestion
```sh
```

File: src/content/docs/workers/tutorials/connect-to-turso-using-workers/index.mdx [hunk 2]
```diff
@@ -128,11 +146,11 @@ The Turso client library requires two pieces of information to make a connection
...
+turso db show my-db --url
+```sh output
 libsql://my-db-<your-github-username>.turso.io
-```
+````
```
Comment (kodster28) [inline src/content/docs/workers/tutorials/connect-to-turso-using-workers/index.mdx]:
```suggestion
```
```

File: src/content/docs/workers/tutorials/connect-to-turso-using-workers/index.mdx [hunk 3]
```diff
@@ -128,11 +146,11 @@ The Turso client library requires two pieces of information to make a connection
 
 To get the URL for your database, run the following Turso CLI command, and copy the result:
 
-```sh
-$ turso db show my-db --url
-# Output:
+````sh
+turso db show my-db --url
+```sh output
```
Comment (kodster28) [inline src/content/docs/workers/tutorials/connect-to-turso-using-workers/index.mdx:153]:
```suggestion
```
```sh output
```

**Comment by isaac-mcfadyen** [comment]
Looks like this PR broke some tables - e.g. see https://developers.cloudflare.com/workers/platform/limits/#worker-limits

**Pull request merged**